script/deploy/shared/mongo-log-utils.ts

```diff
@@ -487,6 +428,9 @@ export class RecordTransformer {
         CONSTRUCTOR_ARGS: string
         VERIFIED: string
         SALT?: string
+        SOLC_VERSION?: string
```

Comment:
here all three are optional. I dont understand why the optionality varies

Reply:
Made all values required everywhere for consistency. Any missing values must be explicitly `""` 874b2566